posthog/models/feature_flag.py

```diff
@@ -77,6 +79,15 @@ def get_filters(self):
             }
 
 
+class FeatureFlagOverride(models.Model):
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["user", "feature_flag"], name="unique feature flag for a user")]
+
+    feature_flag: models.ForeignKey = models.ForeignKey("FeatureFlag", on_delete=models.CASCADE)
+    user: models.ForeignKey = models.ForeignKey("User", on_delete=models.CASCADE)
+    override_value: models.JSONField = models.JSONField(default=bool)
```

Comment:
This is per-user, so what is supposed to happen if a user is in two PostHog projects (`team`s) that use a feature flag with the same key, and switches between from one project to another?